src/components/PromotedActionsBar.tsx

```diff
@@ -41,6 +52,10 @@ function PromotedActionsBar({promotedActions, containerStyle}: PromotedActionsBa
         return null;
     }
 
+    if (promotedActions.length === 0) {
+        return null;
+    }
+
```

Comment:
@kosmydel Any specific reason why the additional identical early return is needed here ?

Reply:
Oh, it must have been added by mistake.